Terasology/LightAndShadow
# Changing player skins through skeletal mesh

Solves [#72](https://github.com/Terasology/LightAndShadow/pull/72)

**How to test**:  
1. Host a Light and Shadow game
2. Join the game using one more client instance
3. Choose one team with each of the players

Player Skin should initially be a white pawn and it should change to a black pawn for the black team, a red pawn for the red team.
